### DISCUSSION TITLE
how can I download save image?

### DISCUSSION THREAD

**Discussion by one0oneChen** [lucidrains/imagen-pytorch#78] [Q&A]
as the title, I has runed the demo.py and everything is ok, but how can I get the image?
![STtJQKu9EG](https://user-images.githubusercontent.com/103432603/175760439-52973ba0-d0da-41dd-b24c-e53454471c65.png)

**Comment by jacobwjs** [comment]
There's only noise to download friend. You need to train first.

**Reply by one0oneChen** [reply]
ohoh,well, thank you my friend. I'll start by learning how to train a model
